keras/layers/embeddings.py

```diff
@@ -50,12 +55,13 @@ def __init__(self, input_dim, output_dim,
                  W_regularizer=None, activity_regularizer=None,
                  W_constraint=None,
                  mask_zero=False,
-                 weights=None, **kwargs):
+                 weights=None, p=0., **kwargs):
```

Comment:
API-wise, `p` does not intuitively make sense in the context of a RNN (although it would in the context of a dropout layer). Prefer using `dropout`.